<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGaugeRange</class>
 <widget class="QWidget" name="frmGaugeRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="GaugeRange" name="gaugeRange1"/>
      </item>
      <item>
       <widget class="GaugeRange" name="gaugeRange2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSlider" name="horizontalSlider1"/>
   </item>
   <item row="0" column="3">
    <widget class="QSlider" name="horizontalSlider2"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugeRange</class>
   <extends>QWidget</extends>
   <header>gaugerange.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
